pelican-gfm/gfm.py

```diff
@@ -29,8 +29,16 @@
 import pelican.utils
 import pelican.signals
 import pelican.readers
-from . import gfmSetup
-from . import Settings
+
+try:
+    from . import gfmSetup
+except ImportError:
+    import gfmSetup
+
+try:
+    from . import Settings
+except:
```

Comment:
If you want to use this approach for imports I think this should only catch an `ImportError`.
See my above comment about other potential approaches.

Reply:
I ended up removing this entirely.